src/App.js

```diff
@@ -21,8 +27,17 @@ const App = () => (
       <Route path="/chat" element={<ChatPage />} />
       <Route path="/keyword" element={<KeywordPage />} />
       <Route path="/mypage" element={<MyPage />} />
+      <Route path="auth/*" element={<AuthPage />}>
+        <Route index element={<AuthMainForm />} />
+        <Route path="register" element={<RegisterDocForm />} />
+        <Route path="registerform" element={<RegisterForm />} />
+        <Route path="idfind" element={<IdFindForm />} />
+        <Route path="pwdfind" element={<PwdFindForm />} />
+        <Route path="pwdchge" element={<PwdChange />} />
```

Comment:
`chge` 는 문맥이 없으면 어떤 단어인지 눈치채기 어렵습니다.
길더라도 명확한 문맥을 쓰는게 좋습니다
`pwd` 는 취향에 따라 갈릴수도 있지만, `chge` 는 확실히 수정하는게 좋겠네요.

```suggestion
        <Route path="pwdChange" element={<PwdChange />} />
```

Reply:
혹시 제가 path쪽은 그냥 다 소문자로만 작성하는데 camelCase로 변경하는게 좋을까요?

Reply:
14f974c